test/system/helpers.bash

```diff
@@ -918,7 +918,7 @@ function skip_if_nocontainer() {
 }
 
 function skip_if_darwin() {
-    [ "$(uname)" == "darwin" ] || skip "Not supported on darwin"
```

Comment:
Darwin is capital on my machine, so this comparison might not work if it's case sensitive

```
$ uname
Darwin
```

Reply:
Yup, I just figured that out.